app/controllers/templates_controller.rb

```diff
@@ -79,13 +79,17 @@ def auto_complete_controller_name
   end
 
   def preview
+    if params[:id]
```

Comment:
@alongoldboim Alright, write a comment explaining it then - and we should be done. Thanks!